runtime/src/main/java/org/capnproto/SerializePacked.java

```diff
@@ -26,19 +26,67 @@
  */
 public final class SerializePacked {
 
+    /**
+     * Attempts to read a message from the provided BufferedInputStream with default options. Returns null if the input
```

Comment:
Would it be more idiomatic to return `Optional<MessageReader>`?

Reply:
Hi David and thanks for the review. I am making the change.